recipe/meta.yaml

```diff
@@ -23,16 +23,15 @@ requirements:
     - toml
     - wheel
   run:
-    - python
-    - colorama  # [win]
-    - importlib_resources  # [py<37]
+    - python >=2.7
```

Comment:
I see you removed it and added it again later. Likely because you got a failure of this test: https://github.com/tqdm/tqdm/blob/master/tests/py37_asyncio.py#L101

test_as_completed is time sensitive so it may fail depending on the load on the builder.
In that case you could just retrigger the build or even skip test_as_completed.

Reply:
Thanks Charles! Your suggestion worked.